### PR TITLE
Marching ants on copy target

### PR DESCRIPTION
**Pull request by davidfig**
Adds a marching ants border around a clipboard copied region. This UI stays until you change something in the sheet or press Escape. (Technically, for the "change something," I'm removing it when I get a `transactionStart` event.)

https://github.com/user-attachments/assets/5ea7333e-21df-4d1c-91d9-6d1f4dbe8e76

### CODE REVIEW

**Comment by luke-quadratic** [comment]
Closes #2112

**Review by ddimaria — APPROVED** [review]
Code looks good

File: quadratic-client/src/app/gridGL/UI/UICopy.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  constructor() {
+    super();
+    events.on('changeSheet', this.updateNextTick);
+    events.on('viewportChanged', this.updateNextTick);
+    events.on('transactionStart', this.clearCopyRanges);
+  }
+
+  destroy() {
+    events.off('changeSheet', this.updateNextTick);
+    events.off('transactionStart', this.clearCopyRanges);
```
Comment (AyushAgrawal-A2) [inline quadratic-client/src/app/gridGL/UI/UICopy.ts:33]: ```events.off('viewportChanged', this.updateNextTick);```

Comment (AyushAgrawal-A2) [reply quadratic-client/src/app/gridGL/UI/UICopy.ts:33]: done

**Review by AyushAgrawal-A2 — APPROVED** [review]

**Pull request merged**